### DISCUSSION TITLE
为什么只能播放480p的视频

### DISCUSSION THREAD

**Discussion by Elen1024** [ywmoyue/biliuwp-lite#97] [Q&A] [answered]
只有两个选项，360p和480p

**Comment by ywmoyue** [comment]
相关issue： https://github.com/ywmoyue/biliuwp-lite/issues/31

暂时没找到原因

**Comment by ywmoyue** [comment, marked as answer]
https://github.com/ywmoyue/biliuwp-lite/discussions/13#discussioncomment-6061866 原因为使用垃圾清理软件清理了Cookie，4.6.9已变更Cookie存储方案
